polls/tests/test_voting.py

```diff
@@ -75,7 +75,7 @@ def test_voting_once_only(self):
         self.assertContains(response, 'is this a test')
         response = client.post(reverse('molo.polls:vote',
                                kwargs={'question_id': question.id}))
-        self.assertContains(response, "You didn&#39;t select a choice")
+        self.assertContains(response, "field is required")
```

Comment:
I had the same feeling about this. @codiebeulaine you can specify a custom error message for the required message in forms.py using [Django's error_messages option](https://docs.djangoproject.com/es/1.9/ref/forms/fields/#error-messages)